src/collective/cover/setuphandlers.py

```diff
@@ -37,18 +27,5 @@ def add_default_layout():
         settings.layouts[default] = EMPTY_LAYOUT
 
 
-def install_relationfield():
-    """Install plone.app.relationfield."""
-    from plone import api
-
-    qi = api.portal.get_tool("portal_quickinstaller")
-    qi.installProduct("plone.app.relationfield")
-    fti = api.portal.get_tool("portal_types")["collective.cover.content"]
-    fti.behaviors += ("plone.app.relationfield.behavior.IRelatedItems",)
-
-
 def run_after(portal_setup):
     add_default_layout()
-
-    if HAS_RELATIONFIELD:
-        install_relationfield()
```

Comment:
relationfield is always installed in plone 5.

Reply:
But you removed the `install_relationfield` function. We can remove the if but keep the function call. It adds a behavior to the cover type. It might be interesting to change the name of the function.

Reply:
`plone.relateditems` is always enables on `collective.cover.content`. See `collective.cover.content.xml`